packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/storage/DatacentreStorage.page.tsx

```diff
@@ -52,6 +57,9 @@ export default function StorageListingPage() {
     'hpc-vmware-managed-vcd/datacentres/compute',
   );
   const navigate = useNavigate();
+  const { data: vcdOrganization, isLoading } = useManagedVcdOrganization({
```

Comment:
error management is missing as well

Reply:
I removed this part since we don't need to check for IAM rights here anymore